src/fespaces/MFEMGenericFESpace.C

```diff
@@ -0,0 +1,37 @@
+#include "MFEMGenericFESpace.h"
+#include "MFEMProblem.h"
+
+registerMooseObject("PlatypusApp", MFEMGenericFESpace);
+
+InputParameters
+MFEMGenericFESpace::validParams()
+{
+  InputParameters params = MFEMFESpace::validParams();
+  params.registerBase("MFEMGenericFESpace");
```

Comment:
No need to register the base class for `MFEMGenericFESpace`, `MFEMScalarFESpace`, `MFEMSimplifiedFESpace`, and `MFEMVectorFESpace`, since they should all inherit from MFEMFESpace anyway; otherwise, each will need to be added to the `add_mfem_fespace` MooseObjectTask in `PlatypusApp.C`

Reply:
Correct

Reply:
I've fixed that in the latest push.